riscv-overlay-software-standard-draft.adoc

```diff
@@ -282,15 +282,15 @@ d. Table shall be aligned with the *_"overlay area"_* so it can be referenced by
 9.  Overlay group size ranges from 512B – 4K for both functions and data
 
 
-NOTE: This table is targeted to be a spec between the running code and the low level driver for loading the overlay function (per group). Since the table is part of the code, the developer can manage it and allocated a placeholder for the overlay groups/functions in the storage for example (storage refers to any software I/F that can fetch the code).
+NOTE: This table is targeted to be a spec between the running code and the low level driver for loading the overlay function (per group). Since the table is part of the code, the developer can manage it and allocated a placeholder for the overlay groups/functions in the storage for example (storage refers to any SW I/F that can fetch the code).
```

Comment:
why adding ```+``` in many places ? 
keep in mind that I use {nbsp}+ for an empty line. Also keep in mind ```NOTE`` is a special word in adoc, it creates "note" logo.
check how github parse it on my branch and yours... also other utilities display it nicely. If you put ```+``` it will not parse it as "NOTE"

Reply:
The `+` and `-` are part of the diff to show what was removed or added. Note that this particular hunk is marked as outdated. If you look at files changed (https://github.com/riscv/riscv-overlay/pull/2/files) rather than individual commits you will see the complete diff master vs my branch as a whole and the note is written as `-NOTE:`.

Reply:
yep, I know... my bad for not explaining myself correctly.... the only issue is with "NOTE"
There is a real "-" there which impact the adoc presentation 

![image](https://user-images.githubusercontent.com/40337637/98521929-be5d3280-227c-11eb-9e65-d04c575705d6.png)

Reply:
Ah I see, the `-` needs to be removed. I probably added that by accident when I was copying from a diff.